### PR TITLE
Pre SDK power update (Java)

### PR DESCRIPTION
**Pull request by stephanie-anderson**
- moved integrations from `/configuration/integrations` to `/integrations` as preparation for the SDK power update
  - added redirects

### CODE REVIEW

File: src/platform-includes/crons/setup/java.mdx [hunk 1]
```diff
@@ -1,4 +1,4 @@
-If you are using [Quartz](http://www.quartz-scheduler.org/), please see <PlatformLink to="/configuration/integrations/quartz/">our Quartz integration</PlatformLink>. If you are using Spring Boot with `@Scheduled` tasks, see [our Spring Boot integration](/platforms/java/guides/spring-boot/crons/).
+If you are using [Quartz](http://www.quartz-scheduler.org/), please see <PlatformLink to="/configuration/quartz/">our Quartz integration</PlatformLink>. If you are using Spring Boot with `@Scheduled` tasks, see [our Spring Boot integration](/platforms/java/guides/spring-boot/crons/).
```
Comment (adinauer) [inline src/platform-includes/crons/setup/java.mdx:1]:
```suggestion
If you are using [Quartz](http://www.quartz-scheduler.org/), please see <PlatformLink to="/integrations/quartz/">our Quartz integration</PlatformLink>. If you are using Spring Boot with `@Scheduled` tasks, see [our Spring Boot integration](/platforms/java/guides/spring-boot/crons/).
```

File: src/platform-includes/crons/setup/java.spring-boot.mdx [hunk 2]
```diff
@@ -1,4 +1,4 @@
-If you are using [Quartz](http://www.quartz-scheduler.org/), please see <PlatformLink to="/configuration/integrations/quartz/">our Quartz integration</PlatformLink>. You may also [send check-ins manually](/platforms/java/crons/).
+If you are using [Quartz](http://www.quartz-scheduler.org/), please see <PlatformLink to="/integrations/quartz/">our Quartz integration</PlatformLink>. You may also [send check-ins manually](/platforms/java/crons/).
```
Comment (adinauer) [inline src/platform-includes/crons/setup/java.spring-boot.mdx:1]: Link 404s because "Integrations" is missing for Spring Boot, maybe others as well.

**Review by adinauer — CHANGES_REQUESTED** [review]
Links seem broken in preview.

**Review by adinauer — APPROVED** [review]
Mostly LGTM, there's a bad link (see failing build) and I'm not sure the apple entries in `vercel.json` were intended.

**Comment by stephanie-anderson** [comment]
@adinauer Thank you for all your feedback! Checked the redirects and moved the last image into its subfolder

**Pull request merged**